mandatory/3-function-output.js

```diff
@@ -1,16 +1,31 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/**
+
+  * @description Function returns a random floating numbers bounded from [0, 10)
+
+ **/
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+/**
+ * @description Function concatenates 2 values together
```

Comment:
This is excellent. You gonna create libraries that we will re-use very soon. Keep it up

Reply:
Thanks a lot bro appreciate it